cloud/IoTHubListener/IoTHubTrigger.cs

```diff
@@ -13,50 +13,26 @@
 
 namespace GWManagementFunctions
```

Comment:
I think we can kill the IoT Hub Triggered version ... even for a toy pattern it's likely not what folks would roll.